packages/trace-viewer/src/ui/uiModeView.tsx

```diff
@@ -177,6 +177,27 @@ export const UIModeView: React.FC<{}> = ({
     });
   }, [projectFilters, runningState, testModel]);
 
+  React.useEffect(() => {
+    const onShortcutEvent = (e: KeyboardEvent) => {
+      if (e.code === 'KeyR' && (e.metaKey || e.ctrlKey)){
+        e.preventDefault();
+        runTests('bounce-if-busy', visibleTestIds);
+      } else if (e.code === 'KeyS' && (e.metaKey || e.ctrlKey)) {
+        e.preventDefault();
+        sendMessageNoReply('stop');
+      } else if (e.code === 'KeyU' && (e.metaKey || e.ctrlKey)) {
+        e.preventDefault();
+        reloadTests();
```

Comment:
I feel like KeyR should be reload which seems more natural to the users. Maybe Enter+Shift should be run?